config/locales/en.yml

```diff
@@ -24,4 +24,6 @@ en:
   activerecord:
     attributes:
       organization:
-        slug: "Organization ID"
\ No newline at end of file
+        slug: "Organization ID"
+      course:
+        custom_points_url: "External scoreboard URL"
```

Comment:
Though not sure whether we should keep on updating the i18n files, as we don't use these at the moment